src/main/java/frc/robot/commands/DefaultArmCommand.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.constants.ArmConstants;
+import frc.robot.constants.JoystickConstants;
+import frc.robot.subsystems.ArmControlSubsystem;
+import frc.robot.subsystems.Input;
+
+public class DefaultArmCommand extends CommandBase {
+  
+  private final ArmControlSubsystem mArmControlSubsystem;
+
+  public DefaultArmCommand(ArmControlSubsystem armControlSubsystem) {
+    mArmControlSubsystem = armControlSubsystem;
+    addRequirements(mArmControlSubsystem);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    handleInput();
+  }
+
+  void handleInput(){
+
+    if(Input.getLeftStickY() > JoystickConstants.PIVOT_INPUT_DEADZONE){
+      mArmControlSubsystem.changeDesiredPivotRotation(JoystickConstants.PIVOT_INPUT_DESATURATION * (Input.getLeftStickY()-JoystickConstants.PIVOT_INPUT_DEADZONE));
+    }else if(Input.getLeftStickY() < -JoystickConstants.PIVOT_INPUT_DEADZONE){
+      mArmControlSubsystem.changeDesiredPivotRotation(JoystickConstants.PIVOT_INPUT_DEADZONE * (Input.getLeftStickY()+JoystickConstants.PIVOT_INPUT_DEADZONE));
+    }
+
+    if(Input.getRightStickY() > JoystickConstants.EXT_INPUT_DEADZONE){
+      mArmControlSubsystem.changeDesiredExtension(JoystickConstants.EXT_INPUT_DESATURATION * (Input.getRightStickY() - JoystickConstants.EXT_INPUT_DEADZONE));
+    }else if(Input.getRightStickY() < -JoystickConstants.EXT_INPUT_DEADZONE){
+      mArmControlSubsystem.changeDesiredExtension(JoystickConstants.EXT_INPUT_DESATURATION * (Input.getRightStickY() + JoystickConstants.EXT_INPUT_DEADZONE));
+    }
+    
+    
+    else if(Input.getA()){
```

Comment:
```suggestion
    if(Input.getLeftStickY() > JoystickConstants.PIVOT_INPUT_DEADZONE){
      mArmControlSubsystem.changeDesiredPivotRotation(JoystickConstants.PIVOT_INPUT_DESATURATION * (Input.getLeftStickY()-JoystickConstants.PIVOT_INPUT_DEADZONE));
      if(Input.getRightStickY() > JoystickConstants.EXT_INPUT_DEADZONE){
        mArmControlSubsystem.changeDesiredExtension(JoystickConstants.EXT_INPUT_DESATURATION * (Input.getRightStickY() - JoystickConstants.EXT_INPUT_DEADZONE));
      }
    }else if(Input.getLeftStickY() < -JoystickConstants.PIVOT_INPUT_DEADZONE){
      mArmControlSubsystem.changeDesiredPivotRotation(JoystickConstants.PIVOT_INPUT_DEADZONE * (Input.getLeftStickY()+JoystickConstants.PIVOT_INPUT_DEADZONE));
      if(Input.getRightStickY() < -JoystickConstants.EXT_INPUT_DEADZONE){
        mArmControlSubsystem.changeDesiredExtension(JoystickConstants.EXT_INPUT_DESATURATION * (Input.getRightStickY() + JoystickConstants.EXT_INPUT_DEADZONE));
      }
    }
    else if(Input.getA()){
```

Reply:
This logic will not work because we want the extension and pivot (represented by the left and right sticks) to be independent of each other. However in the lines you proposed you will only be able to extend outward if you are actively pivoting upward and vise versa if you are pivoting downward.